### PR TITLE
Use appci-ci-operator-credentials instead of apici

### PR DESCRIPTION
**Pull request by hongkailiu**
/cc @smarterclayton @petr-muller 

Clean up after merge:
```bash
oc --context <each_cluster> delete secret -n ci apici-ci-operator-credentials
```

### CODE REVIEW

**Comment by petr-muller** [comment]
In my mental model this makes sense (it seems like we need to do the generator config change first and populate the secret in BW)

**Comment by hongkailiu** [comment]
> In my mental model this makes sense (it seems like we need to do the generator config change first and populate the secret in BW)

It seems that is the other way around. bootstrap change goes first.

**Comment by petr-muller** [comment]
> It seems that is the other way around. bootstrap change goes first.

How does that makes sense? Generator creates a secret in BW, bootstrap reads that secret and puts it to clusters. How can bootstrap be first?

**Comment by hongkailiu** [comment]
> How does that makes sense? Generator creates a secret in BW, bootstrap reads that secret and puts it to clusters. How can bootstrap be first?

I know.
See https://github.com/openshift/release/pull/16399 (it generates a BW item) whose job failed.
https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/16399/pull-ci-openshift-release-master-secret-generator-config-valid/1366810117286137856

I think what the job is doing is to ensure that the generated BW has to be used by bootstrap.

The current PR passed the check as well after https://github.com/openshift/release/compare/644cec213785de6688e21937a6c74964c4b61dc8..48353e3798682e1a95f6589f7fc388b93ec0d1ec

**Comment by petr-muller** [comment]
ok, can we do the secret changes separately so that we get rehearsals?

File: ci-operator/jobs/openshift/release/openshift-release-release-4.1-periodics.yaml [hunk 1]
```diff
@@ -130,12 +130,12 @@ periodics:
         name: release-pull-secret
     serviceAccountName: ci-operator
     volumes:
-    - name: apici-ci-operator-credentials
+    - name: appci-ci-operator-credentials
```
Comment (hongkailiu) [inline ci-operator/jobs/openshift/release/openshift-release-release-4.1-periodics.yaml:133]: Strange.
I cannot use `app.ci-ci-operator-credentials` here. The dot is not allowed in volume name?

**Review by petr-muller — APPROVED** [review]

**Pull request merged**